tsconfig.json

```diff
@@ -4,7 +4,12 @@
     "target": "es6",
     "module": "commonjs",
     "moduleResolution": "node",
+<<<<<<< HEAD
```

Comment:
committed a merge conflict here

Reply:
oops. accepted an overlapping situation when merging that I was going to edit before PR but forgot the editing part. working on it as we speak.